src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -537,8 +538,10 @@ private bool SyncOptionalWorkloads()
                     Dictionary<string, string> owInstalledPkgs = new Dictionary<string, string>();  // packageId -> packageVersion
                     HashSet<string> owSyncRequestsPackageIds = new HashSet<string>();
                     TemplateLocator optionalWorkloadLocator = new TemplateLocator();
+                    Paths paths = new Paths(EnvironmentSettings);
+                    string dotnetPath = Path.GetDirectoryName(Path.GetDirectoryName(paths.Global.BaseDir));
```

Comment:
`dotnet --info` prints a "Base Path". For example:

```
Base Path:   C:\Program Files\dotnet\sdk\5.0.100-preview.7.20366.6\
```

Is this the path we want to be passing to the template locator? If so, what is the reason to not use the same logic that dotnet CLI does, which is simply to call `AppContext.BaseDirectory`? Thank you!

Reply:
What to pass should be `C:\Program Files\dotnet`

that aligns with the environment variable DOTNET_ROOT